src/lib/mongoBackend/dbFieldEncoding.cpp

```diff
@@ -41,8 +41,9 @@
 * Replace:
 *   . => =
 */
-std::string dbDotEncode(std::string s)
+std::string dbDotEncode(const std::string& _s)
```

Comment:
Trying to enforce `const std::string&` as coding style specifies... However, I have needed the `std::string s = s_` ugly statement...

I have also tried `const std::string&` as retun value, but I haven't able to make it work. I have tried the same way `getDbPrefix()` but it doesn't work (maybe in that function it works due to it returns a static string).

In addition, I think it would be a good idea to make this an inline function (moving it to .h). That would make the .cpp be an empty file, but I understand it should be a problem.

Opinions on the above?